src/components/Prescription/Prescription.js

```diff
@@ -35,7 +35,8 @@ const Prescription = () => {
           <p className='to-be-prescribed'>Show to be prescribed:</p>
         </div>
         <p className='script-show-name'>{show.name}</p>
-        <a href={show.officialSite}>Official TV show website</a>
+        {show.officialSite && <a href={show.officialSite} target='_blank'>Official TV show website</a>}
+        {!show.officialSite && <a href={`https://www.google.com/search?q=${show.name}`} target='_blank'>{`Search google for ${show.name}`}</a>}
```

Comment:
Added the target='_blank' so that the links automatically open in a new tab, thus the user can stay on our sweet app